### PR TITLE
fix: footer.tsx

### PR DESCRIPTION
**Pull request by vencecarl15**
footer responsiveness fix

### CODE REVIEW

**Review by LimyuA — APPROVED** [review]
All good but wait for @nchlnnmrvnz 's approval

**Review by nchlnnmrvnz — CHANGES_REQUESTED** [review]
- Center the image
- Missing links for footer? It should include Overview, Hall of Fame, Collection of Projects

**Review by nchlnnmrvnz — APPROVED** [review]

**Pull request merged**
